homeassistant/components/light/scsgate.py

```diff
@@ -0,0 +1,123 @@
+"""
+homeassistant.components.light.scsgate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for SCSGate lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.scsgate/
+"""
+import logging
+import homeassistant.components.scsgate as scsgate
+
+from homeassistant.components.light import Light
+
+from homeassistant.const import ATTR_ENTITY_ID
+
+from scsgate.tasks import ToggleStatusTask
```

Comment:
You should not import requirements in the main module but only inside the methods that actually use them.